.github/workflows/main.yml

```diff
@@ -9,7 +9,7 @@ on:
   workflow_dispatch:
 
 env:
-  VERSION_PREFIX: "1.3.0"
+  VERSION_PREFIX: "1.3.1"
```

Comment:
I think we might also have to bump the version in the `.csproj` file

Reply:
Fixed.